component-models.json

```diff
@@ -214,5 +214,30 @@
         "valueType": "string"
       }
     ]
+  },
+  {
+    "id":"embed",
+    "fields":[
```

Comment:
the order of the fields is different. The picture comes first, the link afterwards but in the same cell. 

see https://main--aem-block-collection--adobe.hlx.live/block-collection/embed

Rather use

- `embed_placeholder`
- `embed_placeholderAlt`
- `embed_uri`

as names